pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -446,24 +453,31 @@ type CertificateAdditionalOutputFormat struct {
 type X509Subject struct {
 	// Organizations to be used on the Certificate.
 	// +optional
+	// +listType=set
```

Comment:
I think ordering might be a bigger issue here.

Reply:
I agree, and will revert to atomic for all subject fields.